torch/_logging/internal.py

```diff
@@ -0,0 +1,398 @@
+# Top level logging module for torch logging
```

Comment:
If the functions in this file should be private, you should make the whole file `_internal`. And define the public APIs in the init directly.
Otherwise everything that doesn't start with `_` in this file will be public and will need to be documented etc

Reply:
If you're happy never exposing anything in the recursive folder. Then it's fine yes.
But if we want to expose things here in the future we want to be a bit more careful.

Reply:
And from all the discussions we're having here. I feel like we do want to have some public API? We do expect users to enable/disable these logs (Dynamo help page recommends enabling some no?)

Reply:
yeah the public API is through an environment variable and torch._logging.set_logs, I did add `set_logs` to the init (and it has a doc block). Should I rename _logging -> logging with the public API in the init, and have all the other internal files prefixed with an underscore? (currently everything is prefixed with an underscore from my latest commit)